src/pages/Home/index.js

```diff
@@ -75,6 +84,28 @@ function Deals({ item }) {
 }
 
 function ProductModal({ selectedId, closeModal, product }) {
+  const store = useCartStore();
+  const cart = store.cart;
```

Comment:
```suggestion
  const { cart, addToCart } = useCartStore();
```

This should also work.

Read more: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment